### ISSUE TITLE
Incorrect hash report after connecting to dev pool

### ISSUE THREAD

**Issue by vegaminer** [fireice-uk/xmr-stak#2384]
1. Is it possible to preconnect to dev pool? It may save time
2. After connecting to dev pool it reports incorrect hashrate

```
==== THIS IS NORMAL REPORT ====
HASHRATE REPORT - AMD
| ID |    10s |    60s |    15m | ID |    10s |    60s |    15m |
|  0 |  926.7 |  926.8 |  921.7 |  1 |  926.3 |  926.5 |  921.7 |
|  2 |  928.3 |  927.8 |  927.7 |  3 |  928.1 |  929.2 |  927.7 |
|  4 |  925.4 |  925.6 |  928.2 |  5 |  928.0 |  927.4 |  928.2 |
|  6 |  925.9 |  926.0 |  927.9 |  7 |  926.2 |  926.0 |  927.9 |
|  8 |  930.8 |  927.6 |  929.6 |  9 |  930.3 |  930.8 |  929.5 |
| 10 |  930.2 |  931.0 |  929.6 | 11 |  930.9 |  931.0 |  929.8 |
| 12 |  921.7 |  923.4 |  926.5 | 13 |  926.0 |  923.4 |  924.5 |
| 14 |  929.9 |  930.0 |  928.7 | 15 |  929.6 |  930.0 |  928.9 |
Totals (AMD):  14844.3 14842.5 14838.0 H/s
-----------------------------------------------------------------
Totals (ALL):   14844.3 14842.5 14838.0 H/s
Highest:  15190.9 H/s
-----------------------------------------------------------------
[2019-03-29 19:16:45] : OpenCL Interleave 2|1: 17/1934.80 ms - 45.0
[2019-03-29 19:16:46] : OpenCL Interleave 3|1: 12/1935.18 ms - 45.0
[2019-03-29 19:16:46] : OpenCL Interleave 6|1: 11/1940.12 ms - 45.0
[2019-03-29 19:16:48] : OpenCL Interleave 4|1: 12/1925.54 ms - 45.0
[2019-03-29 19:16:49] : OpenCL Interleave 2|1: 20/1934.64 ms - 45.0
[2019-03-29 19:16:50] : OpenCL Interleave 3|1: 21/1934.67 ms - 45.0
[2019-03-29 19:16:50] : OpenCL Interleave 7|1: 12/1926.73 ms - 45.0
[2019-03-29 19:16:52] : OpenCL Interleave 4|1: 13/1925.55 ms - 45.0
[2019-03-29 19:16:53] : OpenCL Interleave 2|1: 23/1934.49 ms - 45.0
[2019-03-29 19:16:53] : OpenCL Interleave 3|1: 19/1934.76 ms - 45.0
==== CONNECTING DEV POOL ===
[2019-03-29 19:16:55] : Fast-connecting to dev pool ...
[2019-03-29 19:16:55] : Result accepted by the pool.
[2019-03-29 19:16:55] : Dev pool connected. Logging in...
==== TOOK ONE SECOND TO CONNECT ====
[2019-03-29 19:16:56] : OpenCL Interleave 4|1: 119/1928.61 ms - 45.0
[2019-03-29 19:16:57] : OpenCL Interleave 2|1: 22/1934.58 ms - 45.0
[2019-03-29 19:16:57] : OpenCL Interleave 3|1: 20/1934.78 ms - 45.0
==== REPORT AFTER DEV POOL ====
HASHRATE REPORT - AMD
| ID |    10s |    60s |    15m | ID |    10s |    60s |    15m |
|  0 |   (na) |  952.3 |  922.2 |  1 | 1317.8 | 1028.6 |  928.6 |
|  2 |   (na) |  955.5 |  928.0 |  3 |   (na) |  929.3 |  927.7 |
|  4 | 1326.3 |  394.3 |  893.1 |  5 |   (na) |  947.6 |  928.4 |
|  6 |   (na) |  926.2 |  927.8 |  7 |   (na) |  947.4 |  928.1 |
|  8 |   (na) |  955.1 |  929.9 |  9 |   (na) |  924.4 |  929.5 |
| 10 |   (na) |  961.0 |  930.1 | 11 | 1330.7 |  609.6 |  908.8 |
| 12 |   (na) |  952.9 |  926.9 | 13 | 1318.1 |  810.9 |  917.1 |
| 14 |   (na) |  959.7 |  929.1 | 15 |   (na) |  929.9 |  928.9 |
Totals (AMD):  5292.9 14184.6 14784.0 H/s
-----------------------------------------------------------------
Totals (ALL):   5292.9 14184.6 14784.0 H/s
Highest:  15225.9 H/s
```

**Comment by psychocrypt** [comment]
which currency do you mine?

**Comment by psychocrypt** [comment]
solved in #2387
